frontend/src/components/common/details/details.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import styles from './styles.module.scss';
+
+interface Props {
+  icon: string;
```

Comment:
Create an enum `Icon` add all icons that we use in our application, and by the class, add icon properly